build.sh

```diff
@@ -1,3 +1,2 @@
 rm build/libs/cobblemounts-*.jar
 ./gradlew build
-cp build/libs/cobblemounts-*.jar /home/gavin/.var/app/org.prismlauncher.PrismLauncher/data/PrismLauncher/instances/1.20.1/.minecraft/mods/
```

Comment:
Instead of outright removing this line, please make an `$COBBLEMOUNTS_REPO_OUTPUT` variable which, if non-empty, copies the jar into it. I will then follow up personally by adding the variable to my user's runcom script.